app/controllers/elevator_bookings_controller.rb

```diff
@@ -22,6 +22,7 @@ def create
     render json: { error: "invalid_token" }, status: :unauthorized and return false unless @user
 
     @elevator_booking = ElevatorBooking.new(elevator_booking_params)
+    @elevator_booking.approved = 0
```

Comment:
Just double checking what this is supposed to mean? 
Are all elevator bookings automatically not approved? (It seems that in ruby, all values including zero are evaluated to true).